Principal_component_analysis/include/CGAL/linear_least_squares_fitting_circles_2.h

```diff
@@ -194,7 +194,7 @@ linear_least_squares_fitting_2(InputIterator first,
                    0.0, radius};
     Matrix transformation = init_matrix<FT>(2,delta);
     FT length = 2 * radius;
-    CGAL_assertion(length != 0.0);
+    CGAL_assertion(length != FT(0));
```

Comment:
If we are going to change it, we might as well use `CGAL::is_zero` I guess?

Reply:
Done.